lib/stages/rfiBroadcast.cpp

```diff
@@ -189,7 +190,8 @@ void rfiBroadcast::main_thread() {
                 }
                 // Adjust Stream ID's
                 if (replay) {
-                    StreamIDs[link_id] = link_id;
+                    // TODO: stream_id - this uses internal knowledge of the structure
+                    StreamIDs[link_id].id = link_id;
```

Comment:
Same for a lot of the code here... To make things simpler we might just merge this and then deal with the conflicts in #751 in the interest of getting this PR into develop so it's easier to build on top of.

Reply:
Sounds good to me.